Docs: NoResolution: Hint at how to ignore the issue (#235)
arethetypeswrong/arethetypeswrong.github.io: docs/problems/NoResolution.md

In this example, an import of `"pkg/subpath"` can be used in Node 12+ and modern bundlers, but would fail to resolve in Node 10. Accordingly, TypeScript’s `--moduleResolution node10` (also confusingly known as `node`, because history) does not resolve `"exports"` either.

While few libraries care about supporting a long-past EOL version of Node, maintainers of very popular libraries should be aware that many TypeScript users are still using `--moduleResolution node`. If they aren’t literally using Node 10, they really should move away from it, but sometimes migrating can be difficult because it introduces new errors caused by _incorrect dependency typings_—the problem this tool was made to diagnose. Popular libraries often choose to adopt a strategy that allows `--moduleResolution node10` to work even without `"exports"` support. https://github.com/andrewbranch/example-subpath-exports-ts-compat demonstrates several such strategies.

> ## Ignore issue
> 
> You can use the `profile` `node16` to ignore this issue. It will be reported as `node10: (ignored) 💀 Resolution failed` with a red export path but it the check will not exit with an error anymore. [Learn more in the CLI Readme](https://github.com/arethetypeswrong/arethetypeswrong.github.io/tree/main/packages/cli#profiles).

Reviewer: I’m fine with this addition if it’s rolled into the previous section rather than a new h2. The suggestion isn’t applicable to the problem generally; it’s specific to the `"exports"` cause spelled out above.

```suggestion

> You can use the [`--profile node16` option on the CLI](https://github.com/arethetypeswrong/arethetypeswrong.github.io/tree/main/packages/cli#profiles) to ignore issues that only affect `--moduleResolution node10`.
```
